MetaMask/metamask-extension
# UX: Multichain: Fix AssetListConversionButton Padding

## **Description**

I noticed that the padding in the first implementation of `AssetListConversionButton` was not strict to design -- this PR better pads the element.  

## **Manual testing steps**

1.  Add an account with 0 assets
2. See that the buttons look good

## **Screenshots/Recordings**

### **Before**

<img width="852" alt="SCR-20231011-pcsk" src="https://github.com/MetaMask/metamask-extension/assets/46655/0212dbe5-53aa-448c-bab2-ce1ca8ce6025">


### **After**

<img width="875" alt="SCR-20231011-pcky" src="https://github.com/MetaMask/metamask-extension/assets/46655/fa6233f8-0d53-4dec-8912-75f893600fcb">



## **Related issues**

_Fixes #???_

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
